marko.json

```diff
@@ -89,6 +89,29 @@
         "@style": "string",
         "@disabled": "boolean"
     },
+    "<ebay-combobox>": {
```

Comment:
We'll need an option for autocomplete too. By default a combobox has no autocomplete.